README.md

```diff
@@ -130,12 +130,13 @@ Arguments:
 
 
 #### `cache.save_download`
-The asynchronous process should call this function when the iterable is prepared in order to save
-the output. The only supported format is CSV currently.
+The asynchronous process should call this function when the iterable or file is prepared in order to save
+the output.
 
 Arguments:
 - `download_key`: the cache key of this particular download
 - `iterable`: an iterable of data rows to be written
+- `file`: a BytesIO object
```

Comment:
Could we also indicate that these are optional arguments?

Reply:
I've done a fixup: https://github.com/QuickRelease/django-async-downloads/pull/41/commits/63a3ae5e14b5db5d1add9ad6743d158596c0c1c7